pkg/rt/subscriber.go

```diff
@@ -40,6 +41,7 @@ type Subscriber struct {
 	metrics     *prom.Metrics
 	postprocess func()
 	logger      log.Logger
+	delayCount  int
```

Comment:
`RunRealtime()` and `RunOrigins()` are both called on the same subscriber instance. I think we'll need separate delay counts for origins and billing.

Reply:
ah. good point. fixed in [d4bc0a6](https://github.com/fastly/fastly-exporter/pull/117/commits/d4bc0a62bbf2971157f5dbd27efcbcbc3a34cd1e)